controllers/base_controller.js

```diff
@@ -151,8 +154,26 @@ module.exports = function BaseControllerModule(pb) {
             onlyThisSite: true,
             siteObj: this.siteObj
         };
+        if(pb.config.localization && pb.config.localization.db){
```

Comment:
This entire section of code needs to be moved somewhere else, preferably somewhere we can add responses to a cache so we don't have to call the db on every request to the server.